DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -213,6 +218,45 @@ public override void Initialize()
             Refresh(isHost);
         }
 
+        private void InitializeActivityHandler()
+        {
+            if (activityHandler == null)
+            {
+                activityHandler = new ActivityHandler();
+                activityHandler.PromptInactiveMessage += onPromptInactiveMessage;
+                activityHandler.PromptRemoval += onPromptRemovalFromGame;
+                activityHandler.Start();
+            }
+        }
+
+        private void DisposeOfActivityHandler()
+        {
+            activityHandler.PromptInactiveMessage -= onPromptInactiveMessage;
+            activityHandler.PromptRemoval -= onPromptRemovalFromGame;
+            activityHandler.Dispose();
+            activityHandler = null;
+        }
+
+        public override void OnMouseMove()
+        {
+            base.OnMouseMove();
+
+            if (activityHandler != null)
+            {
+                activityHandler.Reset();
+            }
+        }
+
+        private void onPromptRemovalFromGame(object sender, EventArgs e)
+        {
+            LeaveGame();
+        }
```

Comment:
Methods start with a capital letter. Also, you can make use of the expression body `=>` here.